Makefile

```diff
@@ -100,6 +100,7 @@ ROOT_DIR := $(patsubst %/,%,$(dir $(realpath $(lastword $(MAKEFILE_LIST)))))
 GHC0 ?= ghc-9.8.4
 PYTHON ?= python3
 CABAL ?= cabal
+SED ?= sed
 
```

Comment:
These seem sensible. We should probably extract this change? Seems already like a PR we could merge?